sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -72,6 +72,10 @@ class Configuration
     # RACK_ENV by default.
     attr_reader :environment
 
+    # Whether the SDK should run in the debugging mode. Default is false.
+    # If set to true, the SDK will log every error with full backtrace
```

Comment:
Perhaps here we don't need to promise what debug mode does in such detail ("SDK will log every error with full backtrace"), leaving some possibility to change it later. In any case, we are probably not fully specifying all the consequences of debug mode.